charts/sonarqube/values.yaml

```diff
@@ -362,14 +362,15 @@ annotations: {}
 
 ## We usually don't make specific ressource recommandations, as they are heavily dependend on
 ## The usage of SonarQube and the surrounding infrastructure.
-## Adjust these values to your needs, but make sure that the memory limit is never under 4 GB
+## Adjust these values to your needs, but make sure that the memory limit is never under 6 GB
+## Those default are based on the default Web -Xmx1G -Xms128m and CE -Xmx2G -Xms128m and Search -Xmx2G -Xms2G settings of SQ sub processes
 resources:
   limits:
     cpu: 800m
-    memory: 4Gi
+    memory: 6144M
```

Comment:
Why increase it to 6Gi? Shouldn't we use 5Gi instead?

Reply:
The sum of all XMX is 5Gb and the recommendation is to have XMX at 80% of the total available memory.

This is why we always have a bit more kubernetes request/limit than the sum of all XMX.

Reply:
thanks for clarifying!